### DISCUSSION TITLE
Has the package been updated to work with v2 of the API so we can use GPT-4o?

### DISCUSSION THREAD

**Discussion by geradtroje-env** [openai-php/client#400] [Q&A]
I noticed that when I update the GPT in my assistant from gpt-4-turbo to gpt-4o, it breaks. I read up on this and I think its because you have to use v2 of the API to get that GPT. Is that correct? If so, is that in the works? Just asking :) Thanks in advance!

**Comment by knash94** [comment]
I still need to finish testing all the endpoints and updating the readme but hopefully #395 is stable enough to use now for v2. Within my work projects this has worked fine with gpt4o

**Reply by geradtroje-env** [reply]
Thanks you for replying, Kyle. I will patiently await the next release. Thanks for your time and effort. Can't wait to use gpt-4o.

**Comment by geradtroje-env** [comment]
Closing the thread. Thanks!
